extensions/azurePublish/src/node/luisAndQnA.ts

```diff
@@ -12,8 +12,8 @@ import { AzurePublishErrors } from './utils/errorHandler';
 import {BotProjectDeployLoggerType} from './types';
 
 const readdir: any = promisify(fs.readdir);
-
-const botPath = (projPath: string) => path.join(projPath, 'ComposerDialogs')
+// const botPath = (projPath: string) => path.join(projPath, 'ComposerDialogs')
```

Comment:
kill this if unneeded?

Reply:
I didnt make this change, @benbrown is it okay to remove this? Note changes to this file will affect the existing creation flow

Reply:
I have removed it but will leave this comment to confirm with @benbrown